packages/react-storage/src/components/StorageBrowser/types.ts

```diff
@@ -95,15 +277,29 @@ export type DerivedActionViewType<T extends StorageBrowserActions> =
           : never
         : never]?: any;
     }
-  | Exclude<keyof DefaultActionConfigs, 'download'>;
+  | Exclude<keyof DefaultActionConfigs, DefaultActionWithoutViewType>;
 
+/**
+ * Return values of `createStorageBrowser`
+ */
 export interface CreateStorageBrowserOutput<
   C extends ExtendedActionConfigs = ExtendedActionConfigs,
 > {
+  /**
+   * `StorageBrowser` component and subcomponents
+   */
   StorageBrowser: StorageBrowserType<
     DerivedActionViewType<C>,
     DerivedActionViews<C>
   >;
+
+  /**
+   * Action handler utility hook
```

Comment:
Nit & non-blocking: We can add some example for this one. I plan to show it in the component guidance doc as well.